src/pages/LandingPage/LandingPage.scss

```diff
@@ -18,19 +18,22 @@
   h2 {
     @include dynamic-fonts(2.5em);
   }
+}
 
-  // margin-top: $navbar-height;
-  // @include mobile {
-  //   margin-top: $navbar-height - 50;
-  // }
-
-  // display: flex;
-  // justify-content: center;
-  // align-items: center;
-
-  // .page-header-text {
-  //   font-family: "Roboto";
-  //   color: $color-white;
-  //   @include dynamic-fonts($font-size-header);
-  // }
+.contact-section {
```

Comment:
I've been using ids for things like this since the class probably won't be used for anything else

Reply:
So I think this is a very good idea.

References used for SCSS should be ID's which leaves the className open for use with tailwind.

I've replaced the one mentioned and will continue in the future if we agree here.

Reply:
Sounds good to me.